samples/Common.c

```diff
@@ -704,24 +715,60 @@ VOID sampleBandwidthEstimationHandler(UINT64 customData, DOUBLE maximumBitrate)
 VOID sampleSenderBandwidthEstimationHandler(UINT64 customData, UINT32 txBytes, UINT32 rxBytes, UINT32 txPacketsCnt, UINT32 rxPacketsCnt,
```

Comment:
Can we add a comment here explaining what the sample implementation does at a high level

e.g/ sent bitrate is inversely proportional to the packet loss %